static/admin/config.yml

```diff
@@ -354,3 +357,85 @@ collections:
 
                   - name: "FAQLinkText"
                     widget: "string"
+
+  - name: "technology"
+    label: "Technology"
+    folder: "src/data/technology"
+    create: true
+    delete: false
+    format: "json"
+    identifier_field: "name"
```

Comment:
Shouldn't this be 

`identifier_field: uuid`

Reply:
Ok I see, I think if you did that, the CMS would display the UUID as the title... which would look bad